src/data/data.js

```diff
@@ -0,0 +1,13 @@
+import coinbase from './feed/coinbase';
```

Comment:
Is it worth splitting this into `feed.js` and `random.js` under their respective folders? My first thought was 'why is this file structured differently?'